icons/message-circle.svg

```diff
@@ -9,5 +9,5 @@
   stroke-linecap="round"
   stroke-linejoin="round"
 >
-  <path d="M21 11.5a8.38 8.38 0 0 1-.9 3.8 8.5 8.5 0 0 1-7.6 4.7 8.38 8.38 0 0 1-3.8-.9L3 21l1.9-5.7a8.38 8.38 0 0 1-.9-3.8 8.5 8.5 0 0 1 4.7-7.6 8.38 8.38 0 0 1 3.8-.9h.5a8.48 8.48 0 0 1 8 8v.5z" />
+  <path d="m3 21 1.9-5.7a8.5 8.5 0 1 1 3.8 3.8z"/>
```

Comment:
Neat optimization, but our linter will actually put that ending space back:
```suggestion
  <path d="m3 21 1.9-5.7a8.5 8.5 0 1 1 3.8 3.8z" />
```

Please make sure to replace these ending spaces in all optimization PRs you've submitted. The best course of action would be to make sure that our pre commit hook is properly executed on every change that you submit.